lte/gateway/deploy/roles/agw_docker/templates/env.j2

```diff
@@ -15,6 +15,7 @@ DOCKER_REGISTRY={{ docker_registry }}
 DOCKER_USERNAME={{ docker_user }}
 DOCKER_PASSWORD={{ docker_pass }}
 IMAGE_VERSION={{ image_version }}
+OPTIONAL_ARCH_POSTFIX=
```

Comment:
that should be filled by ansible, right?
```suggestion
OPTIONAL_ARCH_POSTFIX={{ optional_arch_postfix }}
```

Reply:
If we update this then we will have to update Ansible script for adding the value, that change will take more time.

Reply:
I think that might be becoming to hacky and confusing, and it might be worth to go the clean way here

Reply:
Okay, I will update template code.